src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -391,21 +421,27 @@
                 <RowDefinition Height="5*"/>
                 <RowDefinition Height="3*"/>
             </Grid.RowDefinitions>
-
-            <Grid>
+            <local:KeyGraphFeaturesPanel x:Name="KeyGraphFeaturesControl"
```

Comment:
Since this is pretty expensive with all of the richeditboxes, we should consider setting [x:load = false](https://docs.microsoft.com/en-us/windows/uwp/xaml-platform/x-load-attribute)